alerter/src/main/java/com/tapadoo/alerter/Alert.java

```diff
@@ -60,6 +60,7 @@
     private ImageView ivIcon;
     private ViewGroup rlContainer;
     private ProgressBar pbProgress;
+    private boolean dismissable;
```

Comment:
I think this should default to true, as the natural behaviour is that it is dismissible.

Reply:
You're right. It's already changed